src/test_utils.rs

```diff
@@ -111,10 +111,10 @@ impl TestedDialects {
     /// that:
     ///
     /// 1. parsing `sql` results in the same [`Statement`] as parsing
-    /// `canonical`.
```

Comment:
Was this file formatted using `cargo fmt`?

Reply:
One of the fmt warnings was about `canonical` not being adequately spaced, so I changed that.

Reply:
Sure, thank you. I just want to confirm whether those are expected changes in this file.